pkgs/tools/networking/ssh-ident/default.nix

```diff
@@ -10,12 +10,10 @@ stdenv.mkDerivation {
     sha256 = "1jf19lz1gwn7cyp57j8d4zs5bq13iw3kw31m8nvr8h6sib2pf815";
   };
 
-  buildInputs = [ makeWrapper ];
+  buildInputs = [ python3 ];
```

Comment:
Does the executable of interest retain the dependency on the `python3` executable? If not, `wrapProgram` is still needed.

Reply:
looks like the patchShebang is doing the correct behavior